protocols/generic_ngs_prep/fields.json

```diff
@@ -0,0 +1,171 @@
+[
+    {
+      "type": "dropDown",
+      "label": "P20 Multichannel GEN2 Mount Position",
+      "name": "m20_mount",
+      "options": [
+        { "label": "Left", "value": "left" },
+        { "label": "Right", "value": "right" }
+      ]
+    },
+    {
+        "type": "dropDown",
+        "label": "P300 Multichannel GEN2 Mount Position",
+        "name": "m300_mount",
+        "options": [
+          { "label": "Right", "value": "right" },
+          { "label": "Left", "value": "left" }
+        ]
+      },
+    {
+        "type": "dropDown",
```

Comment:
For the temp mod labels here, you should replace letter convention (a/b/c) with the deck slot number to avoid confusion on the users part

Reply:
bump